### ISSUE TITLE
Some issues

### ISSUE THREAD

**Issue by okanoeztuerkbasler** [Alekseon/magento2-widget-forms#14]
We figured out some issues.

(1)

It would be great to set up input elements next to each other in the form settings

Example:
![image](https://user-images.githubusercontent.com/117820640/209802479-53f79a67-3144-4006-b359-23c73112631c.png)

(2)

Copy email under "Confirmation email" does not work

![image](https://user-images.githubusercontent.com/117820640/209807204-6689c888-8934-4831-98fa-4a7fac6af0bc.png)

(3)

New entity notification email is currently only for "global" email addresses, it should be changed to multiple store views

![image](https://user-images.githubusercontent.com/117820640/209807385-8c534685-48c5-47cf-920c-74387803b60f.png)

**Comment by marcinfr** [comment]
Thank You for reporting issues,

(1) we will consider it in the future.

(2) and (3) We fixed these issues in "alekseon/custom-forms-email-notification" version 100.1.6.
You can get updated version by: composer update alekseon/custom-forms-email-notification

Best Regards,

**Comment by okanoeztuerkbasler** [comment]
Thank you, we have downloaded the newest version and (2) und (3) are fixed. :)

We got feedback from customers that the CAPTCHA is always wrong even though they entered the correct characters.
After reloading the CAPTCHA, it works. We can also reproduce it. 

1. Open a form
2. Enter your details
3. Enter CAPTCHA and click Submit
4. Message "Incorrect CAPTCHA" appears
5. Reload CAPTCHA (not the page)
6. Enter new CAPTCHA and click Submit
7. Form is sent successfully
